app/src/data/departments.js

```diff
@@ -61,7 +61,7 @@ const departments = [
   {
     name: 'Emergency Management Services',
     description: 'Emergency systems, communication, and response',
-    learnMore: 'EMS provides emergency communication services, ground and air emergency ambulance response, standby support, and special rescue services to the community.',
+    learnMore: 'The mission of Austin-Travis County EMS is to positively impact and change the lives of the people we serve through the personal commitment of each of our members to provide great patient and customer care, pursue excellence in everything we do, and show compassion to others every day.\n\n EMS provides emergency communication services, ground and air emergency ambulance response, standby support, and special rescue services to the community.',
```

Comment:
I'm wondering if this language make us sound like we are speaking for the City. For ex: `our members`